src/Categories/Category/Instance/Simplex.agda

```diff
@@ -1,67 +1,152 @@
+-- different than the ones presented before, but it's just re-indexing to avoid having to use 'pred'.
 open Category Δ
 
---------------------------------------------------------------------------------
--- Face + Degeneracy Maps
-
-face-map : ∀ {n} → Fin (ℕ.suc n) → Fin n → Fin (ℕ.suc n)
-face-map Fin.zero    k           = Fin.suc k
-face-map (Fin.suc i) Fin.zero    = Fin.zero
-face-map (Fin.suc i) (Fin.suc k) = Fin.suc (face-map i k)
-
-face-mono : ∀ {n} → (i : Fin (ℕ.suc n)) → _≤_ =[ face-map i ]⇒ _≤_
-face-mono Fin.zero    {_}         {_}         le       = s≤s le
-face-mono (Fin.suc i) {Fin.zero}  {_}         _        = z≤n
-face-mono (Fin.suc i) {Fin.suc _} {Fin.suc _} (s≤s le) = s≤s (face-mono i le)
-
-face : ∀ {n} → Fin (ℕ.suc n) → n ⇒ ℕ.suc n
-face i = face-map i , face-mono i
-
-degeneracy-map : ∀ {n} → Fin n → Fin (ℕ.suc n) → Fin n
-degeneracy-map Fin.zero    Fin.zero    = Fin.zero
-degeneracy-map Fin.zero    (Fin.suc k) = k
-degeneracy-map (Fin.suc i) Fin.zero    = Fin.zero
-degeneracy-map (Fin.suc i) (Fin.suc k) = Fin.suc (degeneracy-map i k)
-
-degeneracy-mono : ∀ {n} → (i : Fin n) → _≤_ =[ degeneracy-map i ]⇒ _≤_
-degeneracy-mono Fin.zero    {Fin.zero}  {_}         _        = z≤n
-degeneracy-mono Fin.zero    {Fin.suc _} {Fin.suc _} (s≤s le) = le
-degeneracy-mono (Fin.suc i) {Fin.zero}  {_}         _        = z≤n
-degeneracy-mono (Fin.suc i) {Fin.suc _} {Fin.suc _} (s≤s le) = s≤s (degeneracy-mono i le)
-
-degeneracy : ∀ {n} → Fin n → ℕ.suc n ⇒ n
-degeneracy i = degeneracy-map i , degeneracy-mono i
+-- δᵢ ∘ δⱼ = δⱼ₊₁ ∘ δᵢ if i ≤ j
+face-comm : ∀ {n} {i j : Fin (suc n)}  → i ≤ j → δ (inject₁ i) ∘ δ j ≈ δ (suc j) ∘ δ i
+face-comm {_} {zero}  {j}     z≤n      = Δ-eq refl
+face-comm {_} {suc i} {suc j} (s≤s le) = Δ-eq (λ { {zero} → refl ; {suc x} → cong suc (Δ-pointwise (face-comm le)) })
+
+-- σⱼ ∘ σᵢ = σᵢ ∘ σⱼ₊₁ if i ≤ j
+degen-comm : ∀ {n} {i j : Fin n} → i ≤ j → σ j ∘ σ (inject₁ i) ≈ σ i ∘ σ (suc j)
+degen-comm {_} {zero}  {zero}  z≤n      = Δ-eq λ { {zero} → refl ; {suc x} → refl }
+degen-comm {_} {zero}  {suc j} z≤n      = Δ-eq λ { {zero} → refl ; {suc x} → refl }
+degen-comm {_} {suc i} {suc j} (s≤s le) = Δ-eq (λ { {zero} → refl ; {suc x} → cong suc (Δ-pointwise (degen-comm le)) })
+
+-- σⱼ ∘ δᵢ = δᵢ ∘ σⱼ₋₁ if i < j
+degen-face-comm : ∀ {n} {i : Fin (suc n)} {j : Fin n} → i < suc j → σ (suc j) ∘ δ (inject₁ i) ≈ δ i ∘ σ j
+degen-face-comm {_} {zero}  {j}     (s≤s le) = Δ-eq refl
+degen-face-comm {_} {suc i} {suc j} (s≤s le) = Δ-eq (λ { {zero} → refl ; {suc x} → cong suc (Δ-pointwise (degen-face-comm le)) })
+
+-- σⱼ ∘ δᵢ = id        if i = j
+degen-face-id : ∀ {n} {i j : Fin n} → i ≡ j → σ j ∘ δ (inject₁ i) ≈ id
+degen-face-id {_} {zero}  {zero}  refl = Δ-eq refl
+degen-face-id {_} {suc i} {suc i} refl = Δ-eq (λ { {zero} → refl ; {suc x} → cong suc (Δ-pointwise (degen-face-id {i = i} refl)) })
+
+-- σⱼ ∘ δᵢ = id        if i = j + 1
+degen-face-suc-id : ∀ {n} {i : Fin (suc n)} {j : Fin n} → i ≡ suc j → σ j ∘ δ i ≈ id
```

Comment:
maybe make `j` explicit in the statement here? I think it is more natural to split on explicit arguments.

Reply:
Should we do this with `i` as well, and for every one of these proofs?

Reply:
Any variable that we split on, because we need to know more about it for evaluation to progress, should probably be an explicit argument to these proofs. With a comment that explains why, since it might look like the equational argument might be enough to resolve the implicits.